exercises/rotational-cipher/rotational_cipher.py

```diff
@@ -1,2 +1,2 @@
-def rotate():
+def rotate(string, key):
```

Comment:
Maybe `text` is more appropriate than `string`?

Reply:
Hi @m-a-ge 
I've change the first parameter to 'text'. Please review it again :D